css/style.css

```diff
@@ -1,7 +1,93 @@
+html {
+	box-sizing: border-box; /*padding и border включаются в общую ширину и высоту элемента*/
+}
+
+*, *::before, *::after {
+	box-sizing: inherit;
+}
+
+body {
+	max-width: 1920px; /*резиновая ширина*/
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Project-Bali/pullRequest?prid=5455742)